### ISSUE TITLE
Could you please provide the detailed config and hyperparameters about the action-conditioned model?

### ISSUE THREAD

**Issue by ygtxr1997** [nvidia-cosmos/cosmos-predict2#129]
Amazing work! I'd like to know more training details about the [action-conditioned](https://huggingface.co/nvidia/Cosmos-Predict2-2B-Sample-Action-Conditioned) model listed in huggingface.

Could you please further provide the following details?

- Training config files of the action-conditioned sample model, e.g. a summarized `config.yaml` or training logs
- The number of GPUs and the total training epochs/steps
- The curve of trianing losses

Thank you very much for your help!

**Comment by ygtxr1997** [comment]
Could you please at least provide the `training_steps`, `global batch_size (num_gpus * batch_size_per_gpu)` as the reference?

**Comment by pjannaty** [comment]
Sure!

global batch size: 8 * 32 = 256
learning rate: 2e-5
iterations for convergence for publish 2B model: 30k
